scylla/tests/integration/utils.rs

```diff
@@ -14,6 +19,22 @@ pub(crate) fn setup_tracing() {
         .try_init();
 }
 
+static UNIQUE_COUNTER: AtomicUsize = AtomicUsize::new(0);
+
+pub fn unique_keyspace_name() -> String {
+    let cnt = UNIQUE_COUNTER.fetch_add(1, Ordering::SeqCst);
+    let name = format!(
+        "test_rust_{}_{}",
+        SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_secs(),
+        cnt
+    );
+    println!("Unique name: {}", name);
+    name
+}
```

Comment:
The counter is not shared between `scylla` crate and the integration tests, yet it's a separate counter. Won't there be a name slash between unit tests still using Scylla and integration tests?
Maybe a better solution would be to have this counter exposed in `test_utils` in `scylla` and shared with integration tests?

Reply:
Moving to test_utils in scylla doesn't address this issue at all. Unit tests and integration tests are still 2 separate crates, and don't share memory - so the counters will be separate.

This is however not an issue: those binaries are run sequentially afaik, and current timestamp is included in the name, so there will be no clash.

In the future, the only counter will be in integration tests because unit tests won't connect to database.